client/drum-pad.html

```diff
@@ -1,29 +1,38 @@
-<div ng-controller="drumPad">
+<div class="app-container" ng-controller="drumPad">
```

Comment:
Generally, there is no reason to add a class unless you are going to use it to add css. It doesn't look like this class is being defined is css, so I would just remove it for now. If you're going for readability or browser indexing benefits, you want to do this type of thing: http://www.w3schools.com/html/html5_semantic_elements.asp.